zookeeper-server/src/main/java/org/apache/zookeeper/cli/AclParser.java

```diff
@@ -32,7 +32,6 @@
     /**
      * parse string into list of ACL
      * @param aclString
-     * @return
```

Comment:
These are good, but the `@param aclString` is also not doing anything.

Reply:
- My IDEA doesn't complain about this although the `@param` is really useless
- It's time to do more strict constraint in the `checkstyle-strict.xml `about javaDoc? Are you interested in working for this?

Reply:
BTW, @ctubbsii  Is your github avatar from the [Death Note](https://cn.bing.com/search?q=Death+Note&qs=n&FORM=BESBTB&sp=-1&pq=&sc=0-0&sk=&cvid=A5D97DC314B841378526D876581B4A2E&ensearch=1) ?

Reply:
> BTW, @ctubbsii Is your github avatar from the [Death Note](https://cn.bing.com/search?q=Death+Note&qs=n&FORM=BESBTB&sp=-1&pq=&sc=0-0&sk=&cvid=A5D97DC314B841378526D876581B4A2E&ensearch=1) ?

I think it is "L" lol

Reply:
> * My IDEA doesn't complain about this although the `@param` is really useless

Mine is Eclipse. It doesn't complain by default, but it can find them if you tune the settings. I'm not sure, but Javadoc's built-in `doclint` may also be able to find them.

> BTW, @ctubbsii Is your github avatar from the [Death Note](https://cn.bing.com/search?q=Death+Note&qs=n&FORM=BESBTB&sp=-1&pq=&sc=0-0&sk=&cvid=A5D97DC314B841378526D876581B4A2E&ensearch=1) ?

Of course. :smiley_cat:

Reply:
- Actually, `mvn clean package -DskipTests` can also find the same build warnings, not IDEA. I don't know the mechanism.
- Although this patch has clean up these warnings, if we have no strict constraint on javaDoc. After some time, some codes which have nonstandard javaDocs will be merged again?
- @ctubbsii  @TisonKun Any of you have a free time to do this? :) 
Top priority for javaDoc is: 1.don't allow empty `@param` and empty `@return`; 2. @see and @link must meet specifications.

Reply:
@maoling I'll check if checkstyle rules support guarding what you proposed.

Reply:
@maoling it seems https://checkstyle.sourceforge.io/config_javadoc.html#NonEmptyAtclauseDescription works. Will open a ticket for investigating integrating it.